### DISCUSSION TITLE
CommandLine Args for MacOS?

### DISCUSSION THREAD

**Discussion by netdzynr** [vladmandic/sdnext#1759] [Q&A] [answered]
Following up on [this commandline args topic](https://github.com/vladmandic/automatic/discussions/134), but for MacOS (M1).

I get the following error when when attempting to generate:

`RuntimeError: Input type (c10::Half) and bias type (float) should be the same`

I'm guessing this is due to needing the same commandline args I use in auto1111. Following the posted instructions, I assumed I needed to create `my-webui.sh` with a single line of args as described, but continue to get the same generation error. I also tried creating `my-webui.bat` as well, but no difference. Is a different config needed for MacOS?

Note: torch version is 2.0.0

**Comment by vladmandic** [comment, marked as answer]
which args? check if those even exist here or are moved to settings (as most are).

**Reply by netdzynr** [reply]
Thank you for the pointer.  Found the appropriate settings and generation is working now.
